hermes/src/api/ws.rs

```diff
@@ -108,15 +107,15 @@ impl Subscriber {
             receiver,
             sender,
             price_feeds_with_config: HashMap::new(),
-            ping_interval_future: Box::pin(tokio::time::sleep(PING_INTERVAL_DURATION)),
+            ping_interval: tokio::time::interval(PING_INTERVAL_DURATION),
             responded_to_ping: true, // We start with true so we don't close the connection immediately
         }
     }
 
     pub async fn run(&mut self) {
         while !self.closed {
             if let Err(e) = self.handle_next().await {
-                log::warn!("Subscriber {}: Error handling next message: {}", self.id, e);
+                log::debug!("Subscriber {}: Error handling next message: {}", self.id, e);
```

Comment:
None of the client errors are important and none are actually a problem that we should care. That's why it's changed to debug.